### ISSUE TITLE
Fatal error: Cannot read property 'contents' of undefined

### ISSUE THREAD

**Issue by bedrytskyi** [gruntjs/grunt-contrib-imagemin#365]
Got this when running `grunt`

package.json:
`{
  "name": "sportstatbot",
  "version": "0.1.0",
  "devDependencies": {
    "grunt": "1.0.0",
    "grunt-contrib-concat": "^1.0.1",
    "grunt-contrib-imagemin": "^1.0.1",
    "grunt-contrib-uglify": "^2.0.0",
    "hapi": "*"
  },
  "dependencies": {
    "imagemin": "^5.2.2"
  }
}
`

Gruntfile.js:
`module.exports = function(grunt) {
    grunt.initConfig({
        pkg: grunt.file.readJSON('package.json'),

```
    concat: {
        dist: {
            src: ['check-total-function.js', 'json-from-js.js'],
            dest: 'new-concat.js',
        },
    },

    uglify: {
        build: {
            src: 'new-concat.js',
            dest: 'new-concat.min.js'
        },
    },

    imagemin: {
        dynamic: {
            files: [{
                expand: true,
                cwd: 'images/',
                src: ['**/*.{png,jpg,gif}'],
                dest: 'images/build/'
            }]
        },
    },

});
```

   grunt.loadNpmTasks('grunt-contrib-concat');
   grunt.loadNpmTasks('grunt-contrib-uglify');
   grunt.loadNpmTasks('grunt-contrib-imagemin');

   grunt.registerTask('default', ['concat', 'uglify', 'imagemin']);

   };`

**Comment by bedrytskyi** [comment]
Ok, I changed to `src: ['*/.{png,jpg,gif}'],` but now another problem:

Minified 0 images (saved 0 B)

**Comment by bedrytskyi** [comment]
My folder structure:
├── check-total-function.js
├── Gruntfile.js
├── images
│   ├── 2.jpg
│   ├── build
│   ├── cc643cdfd72634de93598b1c298d92862f6f19e08a162c7ab54ecff32d16ca30.jpg
├── json-from-js.js
├── new-concat.js
├── new-concat.min.js
├── node_modules
├── notes
├── package.json
├── test-json-to-html.html
└── to-html-v2.html

**Comment by bedrytskyi** [comment]
Ok, better go on Gulp)))

**Comment by turbobeast** [comment]
reverting back to `0.8` seemed to fix this for me.

**Comment by kevva** [comment]
Should be fixed in `2.0.0`.
